spec.html

```diff
@@ -11978,7 +11979,7 @@ <h1>Runtime Semantics: GetTemplateObject ( _templateLiteral_ )</h1>
           1. Let _cookedStrings_ be TemplateStrings of _templateLiteral_ with argument *false*.
           1. Let _count_ be the number of elements in the List _cookedStrings_.
           1. Assert: _count_ &le; 2<sup>32</sup> - 1.
-          1. Let _template_ be ! ArrayCreate(_count_).
+          1. Let _template_ be ! ArrayCreate(_count_, %ArrayPrototype%, &laquo; [[TemplateLiteral]] &raquo;).
```

Comment:
This could use a note explaining why this slot is added and not used.

Reply:
Done